opencti-platform/opencti-front/src/private/components/nav/TopBar.tsx

```diff
@@ -292,6 +294,9 @@ const TopBarComponent: FunctionComponent<TopBarProps> = ({
       sub.unsubscribe();
     };
   });
+  useEffect(() => {
+    page();
+  }, [location.pathname]);
```

Comment:
We put the tracker in the TopBar so at every page change we can do a "page view" action, it permits to track the user journey trough the app